python/xgrammar/matcher.py

```diff
@@ -269,7 +270,7 @@ def accept_string(self, input_str: Union[str, bytes], *, debug_print: bool = Fal
         return self._handle.accept_string(input_str, debug_print)
 
     def fill_next_token_bitmask(
-        self, bitmask: torch.Tensor, index: int = 0, *, debug_print: bool = False
+        self, bitmask: ArrayLike, index: int = 0, *, debug_print: bool = False
```

Comment:
What about
```suggestion
        self, bitmask: Union[torch.Tensor, ArrayLike], index: int = 0, *, debug_print: bool = False
```
I still wanna explicitly mention it can accept torch.Tensor because it's the default mask type.